plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -73,13 +75,20 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
               labels: 'labels'
             };
 
+            const networkType = getNetworkType(
+              findNestedPropertyInObject(
+                appDefinition, 'container.docker.network'
+              ),
+              appDefinition
+            );
+
             const containerPortMappings = findNestedPropertyInObject(
               appDefinition, 'container.docker.portMappings'
             );
             if ((portDefinitions == null || portDefinitions.length === 0) &&
```

Comment:
Good catch! I implemented the conditional you provided and it works well. Thanks!